releasePipeline/03-repo-branches-delete.sh

```diff
@@ -90,108 +89,74 @@ function ask_user_for_release_type {
     echo "Chosen type of release process: ${release_type}"
 }
 
+function delete_branches {
 
-function set_kubernetes_context {
-    namespace="galasa-build"
-    h1 "Setting the kubernetes context to be cicsk8s, using namespace ${namespace}"
-    kubectl config set-context cicsk8s --namespace=${namespace}
-    rc=$?
-    if [[ "${rc}" != "0" ]]; then 
-        error "Failed. rc=${rc}"
-        exit 1
-    fi
-    success "Kubernetes context set to cicsk8s using namespace ${namespace}"
-}
+    h1 "Deleting all branches in github called ${release_type}"
 
+    branch_name="${release_type}"
 
+    github_username=$(gh api user --jq '.login')
 
-function delete_branches {
+    if [[ $? != 0 ]]; then
+        error "Failed to get the github username. $?"
+        exit 1
+    fi
 
-    h1 "Deleting all branches in github called ${release_type}"
+    workflow_dispatch=$( gh workflow run branch-delete-all --repo ${github_username}/automation --ref main --field distBranch=${branch_name})
```

Comment:
I don't think the `github_username` variable is required in any of these scripts if this is the only use for it. The release process will always be happening on the galasa-dev repos not forks, so the repo here will always be `galasa-dev/automation`